ci-operator/step-registry/telcov10n/metal-single-node-spoke/gitops/verify/telcov10n-metal-single-node-spoke-gitops-verify-commands.sh

```diff
@@ -37,11 +37,35 @@ function wait_for_managedcluster {
   wait_until_command_is_ok "oc get ns | grep -w '${SPOKE_CLUSTER_NAME}'" 10s 100
 }
 
+function try_to_recover_argocd_clusters_app {
+
+  echo "Cleaning and restoring ArgoCD apps..."
```

Comment:
What is the Argo CD App status that you need to clean?

Reply:
Previous Spoke jobs status. Somethings it takes too long to reconcile.

Reply:
Maybe it's something you can do with `argocd` [cli](https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_app/). Not sure what the issue is...

Reply:
I think this is more or less the same, with out having to install `argocd` cli tool:

https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#force-sync

I have had the chance of reproducing the issue and the solution worked.